v4.14/graph.yaml

```diff
@@ -12,7 +12,9 @@ entries:
   replaces: orchestrator-operator.v1.2.0
 - name: orchestrator-operator.v1.2.3
   replaces: orchestrator-operator.v1.2.1
-name: stable-1.2
+- name: orchestrator-operator.v1.3.0
+  replaces: orchestrator-operator.v1.2.3
```

Comment:
Question: do we want to keep 1.2 and 1.3 separated? maybe we still push fixes for 1.2 in the form of new versions. This change would prevent us from doing so.

Reply:
@pkliczewski WDYT? are we still going to support 1.2 after 1.3 is released?

Reply:
in general the idea is that we follow the support cycle after RHDH. We support for n+1 release and we maintain for n+2

Reply:
do you mean `n-2`?

Reply:
`n` is a release, `n+1` is next release and `n+2` is following release (corresponding 1.2, 1.3 and 1.4). W support and maintain a release for 2 upcoming releases.

Reply:
So in a scenario where`1.4` is current, supported `1.3` and in maintenance (e.g. critical bugs) for `1.2`. Would this be correct?

Reply:
Ok, so now the question is how do we support the upgrade path for our customers: As far as I know, the operator channel dictates the upgrade path. If we use a specific channel for each release, there is no minor upgrade path available, because they will have to change the channel to do that. So they'll have to uninstall the operator, CR included, and create a new subscription that maps to the new channel. Does that work?

Reply:
ok, so they'll have to uninstall the operator to get a new minor release. Only z-stream releases can be installed without requiring wiping out the CR and subscription. `Manual/Automatic` installation plans are be defined in the subscription, which is handled by the customer. Nothing for us to do.